File: executable/r/basic-heatmap.r
library(plotly)

p <- plotly(username='test-runner', key='9h29fe3l0x')

data <- list(
  list(
    z = matrix(c(1, 20, 30, 20, 1, 60, 30, 60, 1), nrow=3, ncol=3), 
    type = "heatmap"
  )
)

response <- p$plotly(data, kwargs=list(filename="basic-heatmap", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: executable/node/basic-heatmap.js
var plotly = require('plotly')('test-runner', '9h29fe3l0x')

var data = [
  {
    z: [[1, 20, 30], [20, 1, 60], [30, 60, 1]], 
    type: "heatmap"
  }
];

plot_url = plotly.plot(data, function (err, msg) {
    console.log(msg);
});